dev/mxcube3/mxcube3-docker-debian9-dev/debian-install.sh

```diff
@@ -49,18 +49,15 @@ install_debian_deps() {
   #ffmpeg
   command apt-get -y install ffmpeg
 
-  #pillow
-  command apt-get -y install libxml2-dev libxslt-dev libtiff-dev libjpeg-dev zlib1g-dev libfreetype6-dev liblcms2-dev libwebp-dev tcl8.5-dev tk8.5-dev python-tk
-
   # npm package imagemin
   command apt-get -y install nodejs nodejs-legacy build-essential nasm libpng12-dev libpng-dev libpng++-dev libpng-tools libpng16-16 zlibc pkg-config
   command apt-get -y install libcairo2-dev libjpeg-dev libpango1.0-dev libgif-dev build-essential librsvg2-dev
-  
+
 }
 
 install_python_deps() {
-  command pip2 install numpy
-  command pip2 install -r requirements.txt
+  command pip3 install numpy
```

Comment:
I guess it makes no difference but this could simply be pip guess ?

Reply:
Analogous to the case above, `pip` is still pointing to pip2 (2.7) in debian 10; for python 3.7 we have to call `pip3`.

Reply:
Ok